src/sql/workbench/browser/modal/modal.ts

```diff
@@ -418,10 +414,10 @@ export abstract class Modal extends Disposable implements IThemable {
 
 
 	/**
-	 * Tasks to perform before dialog is shown
+	 * Tasks to perform before calout dialog is shown
 	 * Includes: positioning of dialog
 	 */
-	protected positionDialog(): void {
+	protected positionCalloutDialog(): void {
```

Comment:
Why did you change this? It's still positioning the normal modal dialog right?

Reply:
This is all my original work, exclusive to callout. I simply renamed it.